bluemix/configuration/core_config/bx_config.go

```diff
@@ -296,11 +296,30 @@ func (c *bxConfig) IAMID() (guid string) {
 	return
 }
 
-func (c *bxConfig) IsLoggedIn() (loggedIn bool) {
-	c.read(func() {
-		loggedIn = c.data.IAMToken != ""
-	})
-	return
+// IsLoggedIn will check if the user is logged in. To determine if the user is logged both the access
```

Comment:
"To determine if the user is logged both the access" ... typo? not sure I follow the language

Reply:
Yeah  its a typo. I meant to say *user is logged in**